src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java

```diff
@@ -5,10 +5,7 @@
 import com.openshift.restclient.IClient;
 import com.openshift.restclient.ResourceKind;
 import com.openshift.restclient.model.IImageStream;
-import hudson.EnvVars;
-import hudson.Extension;
-import hudson.FilePath;
-import hudson.Launcher;
+import hudson.*;
```

Comment:
don't know if you did this or an IDE you are using, but I at least think explicitly listing each class is better than using the package wildcard.

thoughts?